public/css/list/schedule-list.less

```diff
@@ -0,0 +1,99 @@
+// Header
+.schedules-header {
+  margin-left: 12em;
+  width: ~"calc(100% - 12em)";
+
+  .days-header {
+    display: grid;
+    grid-template-columns: repeat(7, minmax(2em, 1fr));
+
+    .column-title {
+      border-right: 1px solid @gray-lighter;
+    }
+
+    .column-title:first-of-type {
+      border-left: 2px solid @gray-lighter
+    }
+  }
+}
+
+// Layout
+.item-list.schedule-list {
+  .list-item {
+    .main {
+      display: flex;
+      align-items: center;
+      header .title {
+        width: 10em;
+        display: inline-flex;
+      }
+
+      .caption {
+        display: flex;
+        margin-left: 0.5em;
+        width: 100%;
+        height: 2em;
+        align-items: center;
+      }
+
+      .zero-rotations {
+        position: absolute;
+        text-align: center;
+        width: 100%;
+        line-height: 1.2;
+      }
+    }
+
+    .timeline {
+      flex-grow: 1;
+      position: relative;
+    }
+
+    .time-grid {
+      grid-template-columns: minmax(0, 1fr);
+      grid-template-rows: minmax(0, 1fr);
+
+      .grid,
+      .overlay {
+        grid-area: ~"3 / 1 / 4 / 3";
+
+        .entry {
+          margin: 0;
+
+          .title {
+            align-items: center;
+          }
+        }
+      }
+    }
+  }
+}
+
+// Design
+.item-list.schedule-list {
+  .zero-rotations {
+    font-size: 1.25em;
+  }
+
+  .time-grid {
+    .grid {
+      border-top: unset;
+      border-left: 2px solid @gray-lighter;
+
+      .step {
+        border-bottom: unset;
+      }
+    }
+  }
+
+  .time-grid:after {
+    border-top: unset;
+  }
+}
+
+// Design
+.item-list.schedule-list {
+  .zero-rotations {
```

Comment:
Everything related to `zero-rotations` should be part of `timeline.less` (including the `position: relative` instruction)